examples/python/monitoring/06_memory_agent_monitoring.py

```
@@ -0,0 +1,218 @@
+        # Retrieve session context
+        if conversation_turn > 1:
+            retrieve_memory(f"session_{session_id}", "short_term")
+        
+        # Search for relevant knowledge
+        search_knowledge_base(query)
+        
+        time.sleep(0.05)  # Simulate context preparation
+    
+    # Main agent execution
+    with track_api_call("memory_agent_execution"):
+        print(f"🚀 Processing query with memory context...")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)